src/TroykaIMU.h

```diff
@@ -13,33 +13,33 @@
 #define GYRO_ADDRESS_V2 L3G4200D_ADDRESS_V2
 
 // Compass
-#define COMPASS_ADDRESS_V1 LIS3MDL_TWI_ADDRESS
-#define COMPASS_ADDRESS_V2 LIS3MDL_TWI_ADDRESS_V2
+#define COMPASS_ADDRESS_V1 LIS3MDL_ADDRESS
+#define COMPASS_ADDRESS_V2 LIS3MDL_ADDRESS_V2
 
 // Barometer
-#define BARO_ADDRESS_V1 LPS331AP_TWI_ADDRESS
-#define BARO_ADDRESS_V2 LPS331AP_TWI_ADDRESS_V2
+#define BARO_ADDRESS_V1 LPS331AP_ADDRESS
+#define BARO_ADDRESS_V2 LPS331AP_ADDRESS_V2
 
 class Accelerometer : public LIS331DLH {
 public:
-    Accelerometer(uint8_t addr = ACCEL_ADDRESS_V1)
-        : LIS331DLH(addr) { }
+    Accelerometer(uint8_t slaveAddress = ACCEL_ADDRESS_V1)
+        : LIS331DLH(slaveAddress) { }
```

Comment:
"Slave" - this is not a buzzword now. When you create a new repository on github, now instead of the `master` branch, the `main` branch is created :)

Reply:
This is data from older commit.
See the latest commit:
https://github.com/amperka/Troyka-IMU/pull/19/commits/66c569ac397a777327247eff8744b723ae126527